internal/environment/aws_environment.go

```diff
@@ -87,15 +87,15 @@ func (a *AWSEnvironment) CPULimit() uint {
 func (a *AWSEnvironment) SetCPULimit(_ uint) {}
 
 func (a *AWSEnvironment) MemoryLimit() uint {
-	panic("not supported")
+	return 0
```

Comment:
Shall we stay with 0 here or switch to 1536 MB according to our deployment (and [analysis](https://lambda-power-tuning.show/#AAIABAAGAAgADA==;3md0RfaQ5EQw0pZEKfRzRBsQNkQ=;5hffN16p0DcPjc43rrHeNwmB+Tc=;AAIABAAGAAgADA==;k/qFRRucAUUsBaZEcV2ERNfTNUQ=;UhMXOJokEjh5eAw4UD4VOODkGTg=;ARM64;x86_64))?

Reply:
Yes, nice :) Currently we use 2048MB that is not cost optimized (as described by the analysis) but with manageable costs of 0,52$ in the last 6 months.
https://github.com/openHPI/poseidon/blob/344dfdfe58fe06b7739c7c65f34b52853f81558b/deploy/aws/template.yaml#L66

```suggestion
	return 2048
```

Reply:
Okay, these costs are really acceptable 😁